src/main/java/com/selfxdsd/todocli/Todo.java

```diff
@@ -167,62 +207,47 @@ public void setPath(final String path) {
      * The objects have the same estimated time;<br>
      * The objects have the same ticket ID.
      *
-     * @param other The object to compare this one to
+     * @param o The object to compare this one to
      * @return True if the objects are equal; false otherwise
      * @checkstyle ReturnCount (50 lines)
      */
     @Override
-    public boolean equals(final Object other) {
-        if (this == other) {
-            return true;
-        }
-        if (other == null || getClass() != other.getClass()) {
-            return false;
-        }
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
 
-        final Todo todo = (Todo) other;
+        Todo todo = (Todo) o;
 
-        if (this.start != todo.start) {
-            return false;
-        }
-        if (this.end != todo.end) {
-            return false;
-        }
-        if (this.estimatedTime != todo.estimatedTime) {
-            return false;
-        }
-        if (!ticketID.equals(todo.ticketID)) {
-            return false;
-        }
-        return true;
+        if (estimatedTime != todo.estimatedTime) return false;
+        if (body != null ? !body.equals(todo.body) : todo.body != null) return false;
+        return ticketID.equals(todo.ticketID);
```

Comment:
@lukacupic Checkstyle will connect about this method: ``other`` instead of ``o``, always use accolades (even if there is only one statement in the ``if``), only one return statement.

Reply:
This is compiler's fault. I also prefer accolades :P